test/module/irohad/ametsuchi/ametsuchi_fixture.hpp

```diff
@@ -18,42 +18,56 @@
 #ifndef IROHA_AMETSUCHI_FIXTURE_HPP
 #define IROHA_AMETSUCHI_FIXTURE_HPP
 
-#include "common/files.hpp"
-#include "logger/logger.hpp"
-
 #include <gtest/gtest.h>
+#include <boost/filesystem.hpp>
 #include <cpp_redis/cpp_redis>
 #include <pqxx/pqxx>
+#include "ametsuchi/config.hpp"
+#include "cli/env-vars.hpp"
+#include "common/files.hpp"
+#include "logger/logger.hpp"
+#include "util/string.hpp"
+
+using namespace std::literals::string_literals;
+using iroha::string::parse_env;
+
+#define LOCALHOST "127.0.0.1"s
+#define ALLHOST "0.0.0.0"s
 
 namespace iroha {
   namespace ametsuchi {
     /**
      * Class with ametsuchi initialization
      */
     class AmetsuchiTest : public ::testing::Test {
-     protected:
-      virtual void SetUp() {
+     public:
+      config::Ametsuchi config;
+
+      AmetsuchiTest(){
         auto log = logger::testLog("AmetsuchiTest");
+      }
+
+      virtual void SetUp() override {
+        config.redis.host = parse_env(IROHA_RDHOST, LOCALHOST);
+        config.redis.port = parse_env(IROHA_RDPORT, 6379);
```

Comment:
Again looks like a good point to move into config.

Reply:
It is "defaults". Config for tests is in env vars.